src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -162,6 +191,26 @@ public void verifyInstructorEditFormDisabled(int instrNum) {
         assertTrue(isNotEditable);
     }
 
+    /**
+     * Verifies that the Instructor Details fields for the given {@code instrNum} contain the updated values.
+     * If {@code newIsDisplayedToStudents} is true, the display name field is checked against {@code newDisplayName}.
+     * Otherwise, it is verified that the display name field's placeholder is shown as expected.
+     */
+    public void verifyInstructorDetails(int instrNum, String newName, String newEmail,
+            Boolean newIsDisplayedToStudents, String newDisplayName, String newRole) {
```

Comment:
@leeyimin A boolean instead of the `void` return type? Not too sure what you meant here. 😅